veracruz-client/src/veracruz_client.rs

```diff
@@ -348,8 +348,10 @@ impl VeracruzClient {
                 let ee_cert = webpki::EndEntityCert::from(certs[0].as_ref())?;
                 let ues = ee_cert.unrecognized_extensions();
                 // check for OUR extension
-                static OUR_EXTENSION_ID: [u8; 3] = [85, 30, 1];
-                match ues.get(&OUR_EXTENSION_ID[..]) {
+                let encoded_extension_id: [u8; 3] = [VERACRUZ_RUNTIME_HASH_EXTENSION_ID[0] * 40 + VERACRUZ_RUNTIME_HASH_EXTENSION_ID[1],
```

Comment:
What are the semantics of `VERACRUZ_RUNTIME_HASH_EXTENSION_ID[0] * 40 + VERACRUZ_RUNTIME_HASH_EXTENSION_ID[1]`?

Reply:
Added a description